tests/api/test_checkout.py

```diff
@@ -1776,3 +1777,218 @@ def test_is_fully_paid_no_payment(checkout_with_item):
+    return """
+        mutation checkoutClearMetadata($token: UUID!, $input: MetaPath!) {
+            checkoutClearMetadata(token: $token, input: $input) {
+                errors {
+                    message
+                }
+            }
+        }
+    """
+
+
+@pytest.fixture
+def clear_checkout_private_meta():
+    return """
+        mutation checkoutClearPrivateMetadata($id: ID!, $input: MetaPath!) {
+            checkoutClearPrivateMetadata(id: $id, input: $input) {
+                errors {
+                    message
+                }
+            }
+        }
+    """
+
+
+@pytest.fixture
+def clear_meta_variables(checkout):
+    return {
+        "token": checkout.token,
+        "input": {"namespace": "test", "clientName": "client1", "key": "foo"},
+    }
+
+
+@pytest.fixture
+def clear_meta_private_variables(checkout):
+    checkout_id = graphene.Node.to_global_id("Checkout", checkout.pk)
+    return {
+        "id": checkout_id,
+        "input": {
+            "namespace": "testPrivate",
+            "clientName": "clientPrivate1",
+            "key": "foo",
+        },
+    }
+
+
+@pytest.fixture
+def checkout_with_meta(checkout):
+    checkout.store_meta(namespace="test", client="client1", item={"foo": "bar"})
+    checkout.store_private_meta(
+        namespace="testPrivate", client="clientPrivate1", item={"foo": "bar"}
+    )
+    checkout.save()
+    return checkout
+
+
+def test_user_without_permission_cannot_clear_private_meta(
+    staff_user,
+    staff_api_client,
+    clear_checkout_private_meta,
+    clear_meta_private_variables,
+):
+    assert not staff_user.has_perm(CheckoutPermissions.MANAGE_CHECKOUTS)
+    response = staff_api_client.post_graphql(
+        clear_checkout_private_meta, clear_meta_private_variables
+    )
+    assert_no_permission(response)
+
+
+def test_staff_user_with_permission_can_clear_meta(
+    staff_user,
+    staff_api_client,
+    clear_checkout_meta,
+    clear_meta_variables,
+    checkout_with_meta,
+    permission_manage_checkouts,
+):
+    staff_user.user_permissions.add(permission_manage_checkouts)
+    assert staff_user.has_perm(CheckoutPermissions.MANAGE_CHECKOUTS)
+    response = staff_api_client.post_graphql(clear_checkout_meta, clear_meta_variables)
+    assert response.status_code == 200
+    content = get_graphql_content(response)
+    errors = content["data"]["checkoutClearMetadata"]["errors"]
+    assert len(errors) == 0
+    checkout_with_meta.refresh_from_db()
+    current_meta = checkout_with_meta.get_meta(namespace="test", client="client1")
+    assert current_meta == {}
+
+
+def test_staff_user_with_permission_can_clear_private_meta(
+    staff_user,
+    staff_api_client,
+    clear_checkout_private_meta,
+    clear_meta_private_variables,
+    checkout_with_meta,
+    permission_manage_checkouts,
+):
+    staff_user.user_permissions.add(permission_manage_checkouts)
+    assert staff_user.has_perm(CheckoutPermissions.MANAGE_CHECKOUTS)
```

Comment:
IMO it is not needed, you just added permission one line above. It is enough meaningful